package.json

```diff
@@ -5,10 +5,10 @@
   "main": "lib/airtap.js",
   "dependencies": {
     "batch": "~0.6.1",
-    "browserify": "~13.3.0",
```

Comment:
Iirc we can't use the latest browserify, so this PR needs tweaking.

Reply:
Why can't we use the latest browserify?

Reply:
Right!

Reply:
So this means browserify no longer supports IE9 and IE10 right? How important is it for us to be able to test on those platforms? I personally don't care at all.

Reply:
Everything but `buffer` supports IE9 and IE10—i think a good thing to do might be to update to a newer browserify version and document how to use the older buffer version if folks do need IE9. i'm switching https://github.com/gozala/events from zuul to airtap and it does run tests in IE9+

Reply:
I'd like to test `memdown` and `level-js` on IE10, because PouchDB targets IE10+.

Also, `readable-stream` targets IE9+ (but isn't updated to use airtap yet) cc @mcollina

Reply:
Btw, there's a workaround for using the version of `buffer` that supports IE9 and IE10 here: https://github.com/browserify/browserify/pull/1678

Reply:
For what it's worth market share for IE9 and IE10 are extremely negligible at this point:

- IE9: 0.12%
- IE10: 0.11%

Even IE8 has more usage, and that hasn't been supported for a long time:

- IE8: 0.19%
- IE11: 2.86%

Source: https://caniuse.com/usage-table

Reply:
If a typed array polyfill makes IE10 work, I'm +1 on updating browserify. But before we proceed, I wanna hear @mcollina's thoughts.

Reply:
I'd like to keep testing `stream-http` on IE9 and IE10 if possible. Would it be possible to add an option somewhere in `airtap` to apply the `buffer` workaround automatically?

Reply:
I'd love to be able to test on IE9 and IE10 but at the same time it kind of sucks to be stuck on a super old version of `browserify`. If we can get both worlds I'd be happy :)

Reply:
Longest discussion on a Greenkeeper PR ever? :wink:

Reply:
> I'd like to test memdown and level-js on IE10, because PouchDB targets IE10+.

:+1:

Reply:
Just some additional context: browserify 14 was released in Jan 2017 (over 1 year ago) and IE9 and IE10 haven't been supported in browserify since then. Perhaps it's still too early to drop support in other libraries (which manage to avoid using `buffer`)

However, I'm curious when folks will feel comfortable dropping support, and what the criteria will be for knowing that it's time?

Reply:
Let's continue the discussion here: https://github.com/airtap/airtap/issues/171